gpbft/participant.go

```diff
@@ -151,6 +151,49 @@ func (p *Participant) ReceiveAlarm() (err error) {
 	return nil
 }
 
+// FinalityInfo is an auxiliary struct to pass relevant info from the finality certificate
+// NOTE: It is not clear to me what information is available in the finality certificate
+// and in what form. The information provided here may change, using it as a placeholder for now.
+type FinalityInfo struct {
+	Instance uint64
+	Power    *PowerTable
+	Beacon   []byte
```

Comment:
I think the instance is all we need. Existing code will fetch the rest when it's needed.

Reply:
Same as above, thanks!